docs/blog/newsletter-august-2017.rst

```diff
@@ -0,0 +1,84 @@
+##################################
+Write the Docs Newsletter – August
+##################################
+
+Hi folks!
+
+This month's issue is coming at you from the sunny Bohemian countryside. I'm in the Czech Republic this week visiting family, but next month, I – and several hundred of you – will back for Write the Docs Prague 2017! Things are getting very real, in conference-planning land, and you can check out what's on the docket for the main stage in `our newly published talk schedule <http://www.writethedocs.org/conf/eu/2017/news/announcing-schedule/>`_!
+
+In other news, Slack's been humming away as usual this month, with so many excellent conversations that it's getting harder and harder to choose just a few to cover here. We've also got nine different local meetups happening around the world this month. So even if you can't join us for the conferences, don't miss out on your other opportunities to interact with the community!
+
+Oh, and also, as I mentioned last month, we're looking to expand our little editorial team for the newsletter. Special thanks to Claire Lundeby (@claire.lundeby), who'll be joining the crew this month! We're still looking for one or two more folks who already spend a lot of time on the Write the Docs Slack, and want to help us turn insightful Slack conversations into mini-features. Drop me a message on Slack @kobrien042, if that sounds like you!
```

Comment:
Probably don't need the qualifier of "spend a lot of time" -- might make people self select out.